core/src/com/unciv/models/ruleset/Building.kt

```diff
@@ -48,6 +48,7 @@ class Building : RulesetStatsObject(), INonPerpetualConstruction {
     var isWonder = false
     var isNationalWonder = false
     fun isAnyWonder() = isWonder || isNationalWonder
+    @Deprecated("Since 3.18.15 - replaced with RequiresAnotherBuilding unique")
```

Comment:
I'm really bad at this whole unique thing but didn't you remove the requiredBuilding instead of deprecating it? Or are lines 252-259 something different? Couldn't find any other depreciation in the commits since 3.18.14 as this is marked `since 3.18.15`